### PR TITLE
Feature/mycredits

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Set the basic functionalities of my-credits.

#### Screenshots or example usage
screen shot 2018-04-30 at 15 24 28

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/locales/en-US.json [hunk 1]
```diff
@@ -279,6 +279,18 @@
...
+  "mycredits.invoices.title": "Invoices",
+  "mycredits.statements.title": "Statements",
+  "mycredits.statements.empty": "No statements yet.",
+  "mycredits.show.invoices": "Show invoices",
+  "mycredits.show.statements": "Show statements",
+  "mycredits.profile.error": "Error while retrieving your account.",
+  "mycredits.nocredit.title": "You do not have any credit available yet.",
+  "mycredits.nocredit.text":
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+  "mycredits.noinvoices": "No future invice to pay",
```
Comment (klzns) [inline react/locales/en-US.json]: typo invoice

File: react/locales/es-AR.json [hunk 2]
```diff
@@ -292,6 +292,18 @@
   "cancel": "Cancelar",
   "export": "Exportar",
 
+  "available.balance": "Saldo disponible",
+  "mycredits.invoices.title": "Títulos",
+  "mycredits.statements.title": "Statements",
```
Comment (klzns) [inline react/locales/es-AR.json]: Faltou traduzir

File: react/locales/pt-BR.json [hunk 3]
```diff
@@ -287,6 +287,18 @@
   "cancel": "Cancelar",
   "export": "Exportar",
 
+  "available.balance": "Saldo disponível",
+  "mycredits.invoices.title": "Títulos de cobrança",
+  "mycredits.statements.title": "Statements",
+  "mycredits.statements.empty": "No statements yet.",
```
Comment (klzns) [inline react/locales/pt-BR.json]: Idem

File: react/locales/en-US.json (hunk 1 above)
Comment (kevinch) [reply react/locales/en-US.json]: Atualizado.

File: react/locales/es-AR.json (hunk 2 above)
Comment (kevinch) [reply react/locales/es-AR.json]: Atualizado.

File: react/locales/pt-BR.json (hunk 3 above)
Comment (kevinch) [reply react/locales/pt-BR.json]: Atualizado.

**Pull request merged**